extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -177,11 +175,11 @@ export class AmpAnalytics extends AMP.BaseElement {
             trigger['selector'] = this.expandTemplate_(trigger['selector'],
                 trigger, /* arg*/ undefined, /* arg */ undefined,
                 /* arg*/ false);
-            addListener(this.win, trigger, this.handleEvent_.bind(this,
+            addListener(this.win.document, trigger, this.handleEvent_.bind(this,
```

Comment:
This should just be now `getAmpDoc()`, right? Why delay migration here? Rather, actually, it'd be a lot better if we called `instrumentation.addEventListener` directly here. I don't see a point in resolving it downstream.